connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConnector.java

```diff
@@ -198,8 +198,7 @@ public Connector connector() {
     @Override
     public String toString() {
         return "WorkerConnector{" +
-                "connName='" + connName + '\'' +
-                ", connector=" + connector +
+                "id=" + connName +
```

Comment:
I was referring not so much to the `toString`, but specifically to its use as the logging prefix. In the producer `TransactionManager`, we use `[TransactionalId Foo]` for example. I don't feel too strongly about it, so we can leave it as is if you prefer.